ref/feature/jaxrsClient/examples.adoc

```diff
@@ -0,0 +1,51 @@
+== Examples
+
+**LC: Could do with an introductory sentence here...**
+
+=== Overriding JAX-RS Client configuration when moving to production
+The JAX-RS Client APIs allow you to specify properties on the client that configure settings like timeouts, proxy
+hosts, SSL properties, and so on. If the application is deployed in a different environment, these settings might need to change. For example, an application might initially be written to run in a development environment, where all remote RESTful services are available without a proxy server and can connect very quickly:
+
+[source,java]
+----
+public Widget getRemoteWidget(String id) {
+    Client client = ClientBuilder.newClient();
+    WebTarget target = client.target("https://somehost:9443/WidgetStore/widgets/");
+    target.property("com.ibm.ws.jaxrs.client.connection.timeout", 5000); // 5 seconds
+    target.property("myCustomProperty", "dev");
+    Widget widget = target.path(id).request().get(Widget.class);
+    return widget;
+}
+----
+
+When this application is deployed in a production environment, however, it might be necessary to modify these settings. This can be accomplished by adding the following configuration to the server config (`server.xml`):
```

Comment:
```suggestion
When this application is deployed in a production environment, however, it might be necessary to add new settings. This can be accomplished by adding the following configuration to the server config (`server.xml`):
```